webapp-django/crashstats/topcrashers/views.py

```diff
@@ -136,6 +138,16 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
+            # Number of startup crashes.
+            hit['startup_percent'] = 0
+
+            sig_startup = hit['facets']['histogram_uptime']
+            for row in sig_startup:
+                if row['term'] == 0:
+                    hit['startup_percent'] = (
```

Comment:
Is there any other purpose of having this variable? 
In the django template, you could ideally move some of the business logic in here in the view code. E.g.
```python
startup_ratio = 1.0 * row['count'] / hit['count']
# here goes some comment that explains why 0.5 (aka 50%) is used and hardcoded in the code. 
hit['startup_crash'] = startup_ratio > 0.5
```